lix/packages/fs/src/utilities/helpers.ts

```diff
@@ -18,19 +18,25 @@ export function assertIsAbsolutePath(path: string) {
 	}
 }
 
-export function normalizePath(path: string, { trailingSlash, leadingSlash }: { trailingSlash?: 'always' | 'strip', leadingSlash?: 'always' } = {}): string {
-	path = path.replace(/^\.\//, '/')
+export function normalizePath(
+	path: string,
+	{
+		trailingSlash,
+		leadingSlash,
+	}: { trailingSlash?: "always" | "strip"; leadingSlash?: "always" } = {}
```

Comment:
Shall we agree on a convention with leading and trailing slashes? Making it configurable seems to be a mistake that needs to be reconsiled with tons of normalization functions. 

If it's a directory, it feels natural to enforce a trailing slash to have a distinction to files. 

I did not find a spec for OPFS https://fs.spec.whatwg.org/#locator-path. Seems like everyone has a different (or no?) spec for path.

Reply:
https://github.com/opral/lix-sdk/issues/30

Reply:
@samuelstroschein  this are only automated changes from prettier, i did not touch anything manually that did not exist before

Reply:
oh lol this is not even in the SDK package ^^